### PR TITLE
Update app-configuration-policies-use-ios.md

### PR DESCRIPTION
**Pull request by msbemba**
Added OnPremisesSamAccountName as tokens used in the property list

Fixes#https://github.com/MicrosoftDocs/memdocs-pr/issues/13357

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/apps/app-configuration-policies-use-ios.md [hunk 1]
```diff
@@ -219,6 +219,7 @@ Additionally, Intune supports the following token types in the property list:
 - \{\{serialnumberlast4digits\}\}—for example, **G5V2** (for iOS/iPadOS devices)
 - \{\{aaddeviceid\}\}—for example, **ab0dc123-45d6-7e89-aabb-cde0a1234b56**
 - \{\{issupervised\}\}—for example, **True** (for iOS/iPadOS devices)
+- \{\{OnPremisesSamAccountName\}\}-for example, **contoso\John**
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/app-configuration-policies-use-ios.md:222]:
```suggestion
- \{\{OnPremisesSamAccountName\}\}—for example, **contoso\John**
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please signoff this PR. Thanks

**Comment by yogkumgit** [comment]
@Erikre Gentle reminder. Thanks

**Comment by Erikre** [comment]
This update is perfect, but to prevent conflicts I'm recreating this PR as a new PR. The changes should be published today. Closing now. Thanks

**Pull request closed without merging**